### DISCUSSION TITLE
Custom emojis for plugins

### DISCUSSION THREAD

**Discussion by larkox** [mattermost/community#59] [Practical Ideas]
Right now there is not a easy way to add custom emojis from a plugin.

Why would we want that?
- Badges! Badges are now emojis, and it would be cool to add the emojis to the system.
- Emoji packs! We can have several "plugins" that are just emoji packs.
- Branding! It make sense that the github plugin adds the octocat emoji, the PR emoji, and stuff like that.

**Comment by esethna** [comment]
Yes please! This is something that was considered in the infrastructure that we implemented for the new emoji updates - so it's certainly possible. Would absolutely love to see emoji packs and plugins that can add emojis!
